docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+# Default Reference Types
+
+We've mentioned throughout this tutorial what the default reference capabilities are for types and classes. However, there's a good chance those defaults passed over your head while you were focused on understanding capabilities themselves.
+
+First, there is nothing saying you have to use default capabilities. Especially as you start writing your first Pony programs, you'll probably find it very educational to explicitly specify the capabilities for all your classes, actors, functions, parameters and fields. However, it's not _writing_ code that makes default capabilities confusing; it's _reading_ that code. Hopefully keeping this summary open will help you navigate pony source code until you've become used to the defaults!
+
+## Reference type summary
+
+We go into detail of the default types below, but here's a table summarizing them:
+
+| Item                    | Default Capability | Explanation                                                             |
+| ----------------------- | ------------------ | ----------------------------------------------------------------------- |
+| primitive               | `val`              | Primitives are immutable and can never be anything but `val`            |
+| actor                   | `tag`              | Actors can neither be read nor written and are always `tag`             |
```

Comment:
Again, each actor has a `ref` reference to itself. Perhaps: Actors are always seen by other actors as `tag`.

Reply:
I think a lot of these explanations while well intended can be misleading and this table overall would be better without the explanations. I'd like to see an attempt made to make the overall content of this page work without the explanation field of the table that I think is ambiguous in places and potentially misleading.

Reply:
I'm leaving the explanations in place for now, but as placeholders. I'm not sure if they can be salvaged or not, but there isn't a lot of sense in updating them until the content is correct.

Reply:
Note: I changed my mind on the editing pass and removed the Explanation column.